barretenberg/ts/src/log.ts

```diff
@@ -0,0 +1,17 @@
+import { ILogObj, Logger } from 'tslog';
+
+export const logger = new Logger<ILogObj>({
+  name: 'bb.js',
+  stylePrettyLogs: false,
+  prettyLogTemplate: '{{dateIsoStr}} {{name}} ',
+  hideLogPositionForProduction: true,
```

Comment:
Add `minLevel: 3` here so we don't print debug logs by default?

Reply:
I was actually going to drop the sublogger to "trace", which is consistent with our old behavior, but same thing yeah